packages/eslint-config-airbnb/README.md

```diff
@@ -41,6 +41,11 @@ If you use yarn, run `npm info "eslint-config-airbnb@latest" peerDependencies` t
   npm install -g install-peerdeps
   install-peerdeps --dev eslint-config-airbnb
   ```
+  
+  or if using node8, use this shortcut
```

Comment:
let's change "node8" to "npm 5+", since that's more accurate.

Reply:
Ok :+1: